rest-api/src/main/resources/logback.xml

```diff
@@ -17,7 +17,7 @@
 			</providers>
 		</encoder>
 	</appender>
-	<root name="gov.cms" level="INFO">
+	<root level="INFO">
```

Comment:
Do we want everything logging at INFO?  Will that output too much framework logging?

Reply:
_I_ don't think it's too noisy for one reason: all the framework logging that does occur is at start-up.  After that, there's nothing except our own application's.

Reply:
agreed with @halprin